common/lib/util/defaults.js

```diff
@@ -116,16 +127,22 @@ Defaults.normaliseOptions = function(options) {
 	if(!('queueMessages' in options))
 		options.queueMessages = true;
 
-	var production = false;
 	if(options.restHost) {
 		options.realtimeHost = options.realtimeHost || options.restHost;
```

Comment:
When options.restHost is specified, realtimeHost is mutated with restHost when realtimeHost is null. Is it possible to have same url for both resthost and realtimehost?

Reply:
> Is it possible to have same url for both resthost and realtimehost?

It is possible yes, and the split between `restHost` and `realtimeHost` is actually now obsolete because they always point at the same endpoint for a given environment. We'll be deprecating them in the spec in the near future.